src/org/omegat/gui/preferences/view/AppearanceController.java

```diff
@@ -40,6 +45,13 @@
 
     private AppearancePreferencesPanel panel;
 
+    private static DefaultComboBoxModel<String> themeSelectionModel = new DefaultComboBoxModel<>();
```

Comment:
It's mostly a convention, but the controller should not retain the model like this. Please see other preference pane examples like `GlossaryPreferencesController.java` where the model is set in `initGui`.

Reply:
Change to set from initGUI().